scene/gui/graph_edit.cpp

```diff
@@ -372,13 +372,13 @@ bool GraphEdit::_filter_input(const Point2 &p_point) {
 
 void GraphEdit::_top_layer_input(const Ref<InputEvent> &p_ev) {
 
-	float grab_r_extend = 2.0;
+	float grab_r_extend = 4.0;
 	Ref<InputEventMouseButton> mb = p_ev;
 	if (mb.is_valid() && mb->get_button_index() == BUTTON_LEFT && mb->is_pressed()) {
 
 		Ref<Texture> port = get_icon("port", "GraphNode");
 		Vector2 mpos(mb->get_position().x, mb->get_position().y);
-		float grab_r = port->get_width() * 0.5 * grab_r_extend;
+		float grab_r = port->get_width() * 0.8f * grab_r_extend;
```

Comment:
this is too messy, just put a radius in pixels as a constant in default_theme and dont use the port texture size

Reply:
Done. Moved the two grab_r variables into the constant. Ditto.